sql/create.sql

```diff
@@ -0,0 +1,10 @@
+create table coupondetails(id serial primary key,
+shopping_site varchar(50),
+discount int not null,
+purchasing_amount int not null,
+start_date date not null,
+expiry_date date not null, 
+status varchar(10));

```

Comment:
Add check constraint for status column e.g: 
CHECK ( status n ('OPEN','CLOSED'))